index.js

```diff
@@ -224,6 +236,38 @@ const fillFieldWithTextWhenVisibleAndPressEnter = function(
   );
 };
 
+const scrollToElement = function(htmlElement)
+{
+  browser.executeScript('arguments[0].scrollIntoView(true);', htmlElement);
+};
+
+const scrollToElementWhenVisible = function(
+  htmlElement,
+  message = getDefaultIsNotVisibleMessage(htmlElement),
+  timeout = DEFAULT_TIMEOUT_IN_MS
+) {
+  this.waitForElementVisibility(htmlElement, message, timeout);
+  this.scrollToElement(htmlElement);
+};
+
+const scrollToElementWhenPresent = function(
+  htmlElement,
+  message = getDefaultIsNotPresentMessage(htmlElement),
+  timeout = DEFAULT_TIMEOUT_IN_MS
+) {
+  this.waitForElementPresence(htmlElement, message, timeout);
+  this.scrollToElement(htmlElement);
+};
+
+const scrollToElementWhenClickable = function(
```

Comment:
I imagine that this function would be used to scroll to an element and right after that click on it. In this case, I don't see the point of having it. I'd suggest leaving only the `scrollToElementWhenVisible` function and then, if you need to click in the element you can use the already existing function `clickWhenClickable`.